user-data/main.tf

```diff
@@ -78,15 +83,17 @@ runcmd:
 - mount -a -t nfs4
 - chmod -R 777 ${var.efs_mount_point}
 EOF
-  }
+
+}
 }
 
 module "teleport_bootstrap_script" {
-  source       = "github.com/skyscrapers/terraform-teleport//teleport-bootstrap-script?ref=3.3.6"
-  auth_server  = "${var.teleport_server}"
-  auth_token   = "${var.teleport_auth_token}"
-  function     = "ecs"
-  environment  = "${var.environment}"
-  project      = "${var.project}"
-  service_type = "upstart"
+source = "github.com/skyscrapers/terraform-teleport//teleport-bootstrap-script?ref=3.3.6"
+auth_server = var.teleport_server
+auth_token = var.teleport_auth_token
+function = "ecs"
+environment = var.environment
+project = var.project
+service_type = "upstart"
```

Comment:
Not part of this PR, but in a next release we should move to Amazon Linux 2 for ECS and thus use the newest teleport modules (with systemd)